.github/ISSUE_TEMPLATE/BUG_REPORT.yaml

```diff
@@ -0,0 +1,43 @@
+name: 🐛 Bug report
+description: Create a report to help us improve 🤔.
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for reporting!  Please also use the search to see if there are any other relevant issues or pull requests.
+
+  - type: input
+    attributes:
+      label: Version
+      description: Please provide either the release version (*e.g.* `0.1.2`) or the first 7 characters of the `git` commit.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What is happening?
+      description: A short description of what is going wrong, in words.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How can we reproduce the issue?
```

Comment:
I saw in other templates that you have this order:
- What is happening?
- How can we reproduce the issue?
- What should happen?

I'm get used to use this other order:
- How can we reproduce the issue?
- What is happening?
- What should happen?

For me it's much more clear the second one but I don't know if you all have agreed previously on using this order because is better for you